src/js/disability-benefits/actions/index.js

```diff
@@ -46,6 +46,22 @@ export function getClaims() {
   };
 }
 
+export function getFilteredClaims(open = true) {
+  return (dispatch) => {
+    makeAuthRequest('/v0/disability_claims',
```

Comment:
Data is only fetched from the server once, the filtering is done client side in this action before being passed to the reducer